botocore/parsers.py

```diff
@@ -535,6 +535,13 @@ def _inject_response_metadata(self, parsed, headers):
             parsed.setdefault('ResponseMetadata', {})['RequestId'] = (
                 headers['x-amzn-requestid'])
 
+    def _parse_body_as_json(self, body_contents):
+        if not body_contents:
+            return {}
```

Comment:
It would be nice to use something different, but I don't really have any other ideas for what we could use instead of `{}`.  I agree that `None` isn't a good option.